### PR TITLE
feat: Udpate Footer

### PR DESCRIPTION
**Pull request by jamesqquick**
Started by @CBID2 💪 

## Description
- add github link to footer
- add community shoutout text to footer
- tweak styling of footer

## Screenshot
CleanShot 2024-08-29 at 11 53 56

## PR Requirements

- [x] I have carefully read through and understand the [Deals-for-Devs Contributing Guide](https://github.com/Learn-Build-Teach/deals-for-devs/blob/dev/.github/contributing.md)
- [x] The title of this PR follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) format
- [x] The `Description` gives a good representation of the changes made
- [x] If this PR addresses an open Issue, it is linked in the `Issue` section

### CODE REVIEW

**Comment by CBID2** [comment]
Thanks for the shout out @jamesqquick

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -37,47 +37,59 @@ const techStack: TechStack[] = [
...
-                id={sponsor.id}
-                size={sponsor.size}
-                className={sponsor.className}
+            {/* clerk was too difficult to make into a sprite */}
+            <a href="https://clerk.com/" target="_blank">
+              <Image
+                src="/icons/clerk.svg"
+                width={159}
+                height={49}
+                className="h-6 w-20 transition-transform hover:scale-105 "
```
Comment (24HourDeveloper) [inline src/components/Footer.tsx:77]: nitpick: Could you remove the space at the end.

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -37,47 +37,59 @@ const techStack: TechStack[] = [
...
-        {/* <Link
-          className="mt-8 text-sm font-medium text-teal-500 md:text-xl"
-          href="https://jamesqquick.com/"
-        >
-          @jamesqquick
-        </Link> */}
+        <span className="text-md flex items-center gap-x-1 font-medium text-white">
+          <span className="">With ❤️ from the</span>
+          <a
+            className=" text-teal-500 hover:text-teal-400"
```
Comment (24HourDeveloper) [inline src/components/Footer.tsx:86]: nitpick: Another space here at the beginning.

**Review by 24HourDeveloper — APPROVED** [review]
Great job!

File: src/components/Footer.tsx (hunk 1 above)
Comment (jamesqquick) [reply src/components/Footer.tsx:77]: Got it!

File: src/components/Footer.tsx (hunk 2 above)
Comment (jamesqquick) [reply src/components/Footer.tsx:86]: Got it!

**Pull request merged**